test/distributed/org/apache/cassandra/distributed/upgrade/UpgradeTestBase.java

```diff
@@ -322,7 +322,6 @@ public void run() throws Throwable
                 try (UpgradeableCluster cluster = init(UpgradeableCluster.create(nodeCount, upgrade.initial, configConsumer, builderConsumer)))
                 {
                     setup.run(cluster);
-
```

Comment:
@Maxwell-Guo put that line back please

Reply:
ok

Reply:
I think you should just rollback the change on this file